test/repository_test.rb

```diff
@@ -50,13 +51,17 @@
             persisted_user.age.must_equal(25)
           end
 
-          it 'assigns and persist created_at attribute' do
+          it 'assigns and persist created_at updated_at attribute' do
             persisted_user = UserRepository.persist(unpersisted_user)
             persisted_user.created_at.must_equal persisted_user.created_at
```

Comment:
on second look, this assertion does not test any valuable information. What do you think about testing `created_at` not nil instead?